pkg/object/httpserver/spec.go

```diff
@@ -93,11 +102,8 @@ func (spec *Spec) Validate() error {
 	}
 
 	if spec.HTTPS {
-		if spec.CertBase64 == "" {
-			return fmt.Errorf("certBase64 is empty when https enabled")
-		}
-		if spec.KeyBase64 == "" {
-			return fmt.Errorf("keyBase64 is empty when https enabled")
+		if spec.CertBase64 == "" && spec.KeyBase64 == "" && spec.Certs == nil && spec.Keys == nil {
+			return fmt.Errorf("certBase64, keyBase64, certs, keys can't be both empty when https enabled")
```

Comment:
We give the direct failure reason instead of `xxx can't yyy`. 
Propose `certBase64/keyBase64, certs/keys are both empty when https enabled`

Reply:
Updated